constants/constants.go

```diff
@@ -0,0 +1,3 @@
+package constants
```

Comment:
Good idea, I think as we go we should continue to look for areas where constants will be helpful.